install_scripts/templates/kubernetes-init.sh

```diff
@@ -384,6 +392,19 @@ if [ -z "$PRIVATE_ADDRESS" ]; then
     promptForPrivateIp
 fi
 
+if [ "$NO_PROXY" != "1" ]; then
+    if [ -z "$PROXY_ADDRESS" ]; then
+        discoverProxy
+    fi
+
+    if [ -z "$PROXY_ADDRESS" ]; then
+        promptForProxy
+    fi
+fi
+
+exportProxy
+# kubeadm requires this in the environment to reach the K8s API server
+export no_proxy="$PRIVATE_ADDRESS,$SERVICE_CIDR" # default service cidr
```

Comment:
I didn't think no_proxy supported CIDR blocks. I guess it probably depends on the implementation.